docs/DafnyRef/UserGuide.md

```diff
@@ -51,129 +96,315 @@ verification failure in a file that is being checked.
 Thus it is important to eventually check all files, preferably in an order
 in which the files without dependencies are checked first, then those that
 depend on them, etc., until all files are checked.
+The `--verify-included-files` option (`-verifyAllModules` in legacy mode) will cause all modules, whether the result of include directives or not,
```

Comment:
I thought that while users are in transition mode, it would be helpful to be told the old/new names where there are obvious pairs. After the new CLI is well-established, I would remove these references.